keras_nlp/layers/masked_lm_mask_generator.py

```diff
@@ -147,11 +147,7 @@ def call(self, inputs):
             # convert dense to ragged.
             inputs = tf.RaggedTensor.from_tensor(inputs)
 
-        (
-            token_ids,
-            mask_positions,
-            mask_ids,
-        ) = tf_text.mask_language_model(
+        (token_ids, mask_positions, mask_ids,) = tf_text.mask_language_model(
```

Comment:
This formatting is due to a black version change. Could you upgrade black and rerun the formatting? Thanks!